### DISCUSSION TITLE
Tabs with dropdown buttons

### DISCUSSION THREAD

**Discussion by kewartoscar** [zauberzeug/nicegui#3076] [Q&A] [answered]
### Question

Is it possible to make tabs dropdown buttons such that each dropdown item is linked to a page?

**Comment by falkoschindler** [comment]
Sorry, @kewartoscar, I don't understand what you mean.

**Reply by kewartoscar** [reply]
![dropdown1](https://github.com/zauberzeug/nicegui/assets/63175395/0ea87557-ff11-4cbc-a1b3-19b640094836)
![dropdown2](https://github.com/zauberzeug/nicegui/assets/63175395/ac93356c-43d7-40d8-bf44-1041cb6f0418)

A tab with a dropdown menu that has items which when clicked leads to different pages.
For instance, When the Software Categories is clicked, there is a dropdown menu with items:
CRM Software, Email marketing Software, Help Desk Software, Human Resource Software, Project management Software, and Browse All Categories. When any of these items are clicked, it takes one to a specific page.

**Comment by falkoschindler** [comment, marked as answer]
I see. Here's an example using a `ui.dropdown_button`:

```py
with ui.tabs() as tabs:
    one = ui.tab('One')
    two = ui.tab('Two')
    with ui.dropdown_button('More', auto_close=True).props('flat square color=black').classes('h-full'):
        ui.item('Three', on_click=lambda: tabs.set_value('three'))
        ui.item('Four', on_click=lambda: ui.open('/four'))
with ui.tab_panels(tabs, value=one).classes('w-full'):
    with ui.tab_panel(one):
        ui.label('First tab')
    with ui.tab_panel(two):
        ui.label('Second tab')
    with ui.tab_panel('three'):
        ui.label('Third tab')
```
<img width="271" alt="Screenshot 2024-05-16 at 08 33 36" src="https://github.com/zauberzeug/nicegui/assets/5767091/117111fe-17ad-4b58-8482-666c6957261c">

I wasn't sure if you want to open pages or tabs, so I'm showing both.

**Comment by kewartoscar** [comment]
Thanks a million. This is exactly what I was looking for.
